### ISSUE TITLE
[BUG] Items disappearing from Finish Watching movies list

### ISSUE THREAD

**Issue by Navios92** [nixgates/plugin.video.seren#355]
I'm not sure how to properly capture a log for this issue. Movies sometimes randomly disappear from the Finish Watching list. Restarting Kodi or simply playing a movie can trigger this.

I can confirm it's not a trakt problem as when I check [here](https://sharkykh.github.io/tppm/), I can see all the movies with correct resume points.

**Comment by Navios92** [comment]
There is another problem with Finish Watching. It doesn't seem to be working across different devices. Movies just restart from the beginning or get the resume point from the device itself, not trakt.

**Comment by Bulletbling** [comment]
I’ve had this issue too. Movies randomly disappear from finish watching list, but for me it’s usually after starting to play the movie and stopping it. The widget then no longer displays the movie and a Kodi restart fixes it. Don’t know if this is the OP issue as well though.

**Comment by nixgates** [comment]
From what I can tell, going of the fact that the item is there after a restart (I'm almost certain it would be there after a refresh of that list as well) the local resume point is created after the widget is called. Seren updates the local resume point after the playback is stopped. 

I'll investigate it, but it might require continual updates from Seren to the local database to create the resume point. I'll have to investigate.

**Comment by Bulletbling** [comment]
Maybe we will have to re enable widget refresh option after movie closeout. It's a nice feature to be able to turn off however, there has to be a local way to keep Scrabble progress and it's updated internally with seren before it's able to be sent to trakt, which sometimes doesn't always get get retrieved successfully.

Not sure if this is related, but when I use the venom add-on and it scrabbles to trackt (confirmed in upper right corner notification, and then go into the seren video add-on app and go to my movies and continue watching and nothing shows up there.

**Comment by nixgates** [comment]
@Bulletbling you'll need to remember that items scrobbled outside of Seren will take up to 15 minutes to roll over into Seren.

**Comment by Navios92** [comment]
> @Bulletbling you'll need to remember that items scrobbled outside of Seren will take up to 15 minutes to roll over into Seren.

That wasn't always the case, right? I do remember that in the previous version of Seren, local library movies would appear in Seren's Finish Watching right after playback stops.

**Comment by Bulletbling** [comment]
> > @Bulletbling you'll need to remember that items scrobbled outside of Seren will take up to 15 minutes to roll over into Seren.
> 
> That wasn't always the case, right? I do remember that in the previous version of Seren, local library movies would appear in Seren's Finish Watching right after playback stops.

I thought this was the case as well, but that was prior to being able to have those option of library addon widgets refresh enabled or disabled.m, which has been implemented in 2.0 iirc. It just sucks that even after closing a seren movie, the movie still won't show up and continue watching either ever, or sometimes it is a delay. I'd rather have the movie I just stopped watching before to show in continue watching section and then and update that removes it if I had completed the movie.
